provider/script.go

```diff
@@ -92,6 +106,11 @@ func (e *Script) StringGetter() func() (string, error) {
 		if time.Since(e.updated) > e.cache {
 			e.val, e.err = e.exec(e.script)
 			e.updated = time.Now()
+
+			if e.jq != nil {
+				v, err := jq.Query(e.jq, []byte(e.val))
+				return fmt.Sprintf("%v", v), err
```

Comment:
Du gibst etwas zurück das nicht im cache steht

Reply:
Ah ok.
Bin mir jetzt unsicher, ob ich, falls bei jq.Query ein Fehler auftritt, diesen in den Cache schreiben und zurückgeben soll, oder ob der Cache dann unverändert bleibt und ich nur den Query-Fehler zurück geben soll.
https://github.com/Alestrix/evcc/blob/6071f4de29cb2a48e63e0bcc85a1f03c2010c727/provider/script.go#L111-L116

Reply:
...achich Dödel, muss val ja auch noch in den Cache schreiben...